src/shared/lightbox.scss

```diff
@@ -10,7 +10,7 @@
 	}
 
 	.components-modal__header .components-button:hover {
-		color: var( --wp--preset--color--white );
+		color: #666;
```

Comment:
this was added, because of day/night mode.

Reply:
What would be the best way for me to test that?

Reply:
a theme that supports light/dark mode and switch between both modes to see if it works in both.

Reply:
https://github.com/pfefferle/autonomie ☺️

but this may not be the best one... there might be better ones... 🫣

Reply:
See https://github.com/Automattic/wordpress-activitypub/issues/721 and https://github.com/Automattic/wordpress-activitypub/issues/425

Reply:
Fixed in [29db1be](https://github.com/Automattic/wordpress-activitypub/pull/1034/commits/29db1be3fad07740efd329f0183b05ecd8813f1f). Frankly, I'm a bit confused by it because the background and text colors of the modal don't change, but the hover for the button does.

Even with the changes it doesn't look great in dark mode, but at least it doesn't disappear now.